### PR TITLE
Imhimansu28 patch 1

### PR DESCRIPTION
**Pull request by imhimansu28**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3019"></a>

### CODE REVIEW

File: himanshu.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+
+{
+  "name": "Himanshu Gupta",
+  "type": "personal",
```
Comment (ChinmayMhatre) [inline himanshu.json:5]: Please remove this file from the PR.

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Please rename the json file in the data directory to your github username. Remove the one that is not in the data folder

File: himanshu.json (hunk 1 above)
Comment (imhimansu28) [reply himanshu.json]: ok Done

File: data/himanshu.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
+{
+  "name": "Himanshu Gupta",
+  "type": "personal",
+  "bio": "I am Software Developer in Mittal & Co. Pvt. Ltd.",
+  "avatar": "https://github.com/imhimansu28.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/imhimansu28/",
```
Comment (ChinmayMhatre) [inline data/himanshu.json:10]: Rename the file to imhimansu28.json

**Review by ChinmayMhatre — APPROVED** [review]
left some comments

**Review by ChinmayMhatre — APPROVED** [review]
LGTM!

File: data/imhimansu28.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/imhimansu28",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/himanshu-gupta-033793188/",
+      "icon": "FaLinkedin"
+    }
+  ],
```
Comment (swagataroy30) [inline data/imhimansu28.json:23]: Remove the comma on line 23.

**Comment by ChinmayMhatre** [comment]
Has this been merged in some other PR? @imhimansu28

**Comment by imhimansu28** [comment]
Under review

**Pull request closed without merging**